.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+ECJ_VERSION="4.6.1"
+
+
+mkdir target/classes
+mkdir target/eclipse
+
+ECJ_EXISTS="$(ls ~/.m2/repository/ | grep ecj-$ECJ_VERSION.jar)"
```

Comment:
there should not be any grep, file should be located in certain place - "~/.m2/repository/ecj-$ECJ_VERSION.jar"